applications/DEMApplication/python_scripts/DEM_procedures.py

```diff
@@ -1034,6 +1038,17 @@ def FinalizeGraphs(self, rigid_face_model_part):
         if self.TestType == "None":
             self.close_graph_files(rigid_face_model_part)
 
+    def PrintAdditionalGraphs(self, time, solver):
```

Comment:
Let's check that we are not printing graphs always with this function. Printing should always be an option marked by the user, not hardcoded and always being called...

Reply:
Yes, it only prints the coordination number only if the user ask for it.